bin/radical-pilot-hello.sh

```diff
@@ -48,12 +48,18 @@ test -z "$GPU_INFO" && GPU_INFO="$CUDA_VISIBLE_DEVICES"
 test -z "$GPU_INFO" && GPU_INFO="$GPU_DEVICE_ORDINAL"
 GPU_INFO=$(echo " $GPU_INFO " | tr ',' ' ')
 
-LSPCI=$(which lspci 2> /dev/null)
-test -z "$LSPCI" && LSPCI='/sbin/lspci'
-test -f "$LSPCI" || LSPCI='/usr/sbin/lspci'
-test -f "$LSPCI" || LSPCI='true'
-GPU_NBITS=$($LSPCI | grep -e " VGA " -e ' GV100GL ' | wc -l)
-
+UID=$(id -u)
+LSPCI_CACHE="/tmp/lspci.$UID"
```

Comment:
@andre-merzky, seems like better to use a different name for `UID` (e.g., `USER_ID`)
with tests on Summit got the following error
```
./radical-pilot-hello.sh: line 51: UID: readonly variable
```

Reply:
Right, I fixed that.  Thanks.